### PR TITLE
add doc about how to use grafana to visualize service metrics

### PR DESCRIPTION
**Pull request by YaoZengzeng**
add doc about how to use grafana to visualize service metrics

fix part of: https://github.com/kmesh-net/kmesh/issues/600 , L7 Observability Metrics

### CODE REVIEW

File: content/en/docs/userguide/service_metrics.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+  docs:
+    parent: user guide
+    weight: 2
+title: Use Grafana to visualize service metrics
+toc: true
+type: docs
+
+---
+
+### Preparation
```
Comment (hzxuzhonghu) [inline content/en/docs/userguide/service_metrics.md:14]: kind of duplicate with https://kmesh.net/en/docs/userguide/try_waypoint/

File: content/en/docs/userguide/service_metrics.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+[root@ ~]# kubectl port-forward --address 0.0.0.0 svc/grafana 3000:3000 -n kmesh-system
+Forwarding from 0.0.0.0:3000 -> 3000
+```
+
+2. View the dashboard from browser
+
+Visit `Dashboards>Kmesh>Kmesh Service Dashboard`:
+
+<div align="center">
+<img src="/docs/userguide/pics/grafana.png" width="1400" />
```
Comment (hzxuzhonghu) [inline content/en/docs/userguide/service_metrics.md:80]: Cannot render it out

File: content/en/docs/userguide/service_metrics.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+toc: true
+type: docs
+
+---
+
+### Preparation
+
+Refer [Install-kmesh | Quick Start | Kmesh](https://kmesh.net/en/docs/setup/quickstart/#install-kmesh)
+
+- install Kmesh,
```
Comment (hzxuzhonghu) [inline content/en/docs/userguide/service_metrics.md:18]: why is this  smaller than main body

File: content/en/docs/userguide/service_metrics.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+    weight: 2
+title: Use Grafana to visualize service metrics
+toc: true
+type: docs
+
+---
+
+### Preparation
+
+Refer [Install-kmesh | Quick Start | Kmesh](https://kmesh.net/en/docs/setup/quickstart/#install-kmesh)
```
Comment (hzxuzhonghu) [inline content/en/docs/userguide/service_metrics.md:16]: This is redudant as has been included in install waypoint

**Comment by YaoZengzeng** [comment]
@hzxuzhonghu updated

**Review by hzxuzhonghu — APPROVED** [review]
/lgtm

**Pull request merged**